tests/bookmarks_and_history/test_delete_other_bookmars.py

```diff
@@ -0,0 +1,51 @@
+import sys
+from os import environ
+
+import pytest
+from selenium.webdriver import Firefox
+
+from modules.browser_object import ContextMenu, Navigation
+from modules.classes.bookmark import Bookmark
+from modules.page_object import GenericPage
+from modules.util import BrowserActions
+
+
+BOOKMARK_URL = "about:robots"
+BOOKMARK_URL_2 = "about:cache"
+
+
+WIN_GHA = environ.get("GITHUB_ACTIONS") == "true" and sys.platform.startswith("win")
+
+
+@pytest.mark.skipif(WIN_GHA, reason="Test unstable in Windows Github Actions")
+def test_delete_other_bookmarks(driver: Firefox):
+    """
+        C2084524: Verify that a user can Delete a bookmark from 'Other Bookmarks' folder
```

Comment:
These days we need to indicate the test case in a fixture. Outside of this test function, please write a fixture function:

```python
@pytest.fixture()
def test_case():
    return "2084524"
```

Reply:
Fixture added